### PR TITLE
OR-1182: Creates activity group page for detailed desc prototype

### PR DESCRIPTION
**Pull request by mangogrin**

### CODE REVIEW

**Comment by chilliwilly77** [comment]
:+1

File: app/locales/en.json [hunk 1]
```diff
@@ -172,4 +199,4 @@
     "404": "Page not found",
     "500": "We're experiencing technical problems."
   }
-}
+}
```
Comment (chilliwilly77) [inline app/locales/en.json]: blank line required at end of file

File: test/common/page_objects/page.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+class Page {
```
Comment (chilliwilly77) [inline test/common/page_objects/page.js]: Do you use this anywhere?

Comment (mangogrin) [reply test/common/page_objects/page.js]: no, removed

File: app/locales/en.json (hunk 1 above)
Comment (mangogrin) [reply app/locales/en.json]: done

File: app/controllers/activity-group-view-model.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+module.exports = class ActivityGroupViewModel {
+
+  constructor(activityGroups) {
+    this.activityGroups = this.activityGroupView(activityGroups);
+  }
+
+  activityGroupView(activityGroups) {
+    return activityGroups
+      .map((activityGroup) => Object.assign(
```
[the remark this answers is not in the document]
Comment (mangogrin) [reply app/controllers/activity-group-view-model.js]: done

**Pull request merged**